README.md

```diff
@@ -107,4 +107,8 @@ config dir: /home/tj/.pyenv/versions/3.8.1/envs/general/etc/jupyter
       jupyter_ascending 0.1.13 OK
 ```
 
-Run tests from the root directory of this repository using `python -m pytest .`.
\ No newline at end of file
+Run tests from the root directory of this repository using `python -m pytest .`.
+
+Format files with `pyfixfmt`. In a PyCharm file watcher, something like
+
+```python -m pyfixfmt --file-glob $FilePathRelativeToProjectRoot$ --verbose```
```

Comment:
I think pyfixfmt is our own code right now. We should really probably make a separate repository for that on github... or at least not depend on it anywhere!

Reply:
It's on PyPI! https://pypi.org/project/pyfixfmt. And it's not a dependency anywhere in the code, just for developers looking to match the formatting style. Agree we should break it out into a separate repo though.

Reply:
Ok, I went ahead and created https://github.com/untitled-ai/pyfixfmt (still private, made some issues in there for todos).

Reply:
Great! Answered a few of the issues over there. Thanks for splitting it out